runbooks/key-handover/VALIDATION.md

```diff
@@ -0,0 +1,57 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of Key Handover process. 
```

Comment:
Not sure we want to be duplicating this `validation.md` file. 

We have the mmz one here which can be reused: https://github.com/ethereum-optimism/superchain-ops/blob/main/tasks/eth/mmz-002-key-handover/VALIDATION.md